### PR TITLE
Feature: Move examples/dao scripts and test to SigmaDAO repo with CI enabled

### PR DESCRIPTION
**Pull request by rmeena840**

### CODE REVIEW

**Comment by rmeena840** [comment]
@Megha-Dev-19 @robert-zaremba Please review the PR. Thanks

File: contracts/test/common.js [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+		this.proposalLsigAcc = this.getAccount(this.proposalLsigAcc.address);
+	}
+
+	deployASA(name, creator) {
+		this.govTokenID = this.runtime.deployASA(name, { creator: creator }).assetIndex;
+	}
+
+	deployDAOApp(sender, daoApprovalProgramFileName, daoClearStateProgramFileName) {
+		// const daoApprovalProgram = getProgram(approvalProgram, , false);
+		// const daoClearProgram = getProgram(clearStateProgram, {}, false);
```
Comment (Megha-Dev-19) [inline contracts/test/common.js]: if we don't need it, can we remove it

File: dapp/package.json [hunk 2]
```diff
@@ -22,8 +22,8 @@
 		"upgrade:dependency": "npx yarn-upgrade-all"
 	},
 	"dependencies": {
-		"@algo-builder/algob": "^5.0.1",
-		"@algo-builder/web": "^5.0.1",
+		"@algo-builder/algob": "^6.0.0",
+		"@algo-builder/web": "^6.0.0",
```
Comment (Megha-Dev-19) [inline dapp/package.json:26]: have you tested the dapp after upgrading the web package, to see if there are any breaking changes?

File: infrastructure/private-net-template.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{
+	"Genesis": {
+		"NetworkName": "devnet",
```
Comment (Megha-Dev-19) [inline infrastructure/private-net-template.json:3]: shouldn't it be privatenet?

File: contracts/test/common.js (hunk 1 above)
Comment (rmeena840) [reply contracts/test/common.js]: done

File: dapp/package.json (hunk 2 above)
Comment (rmeena840) [reply dapp/package.json:26]: Yup. All working

File: infrastructure/private-net-template.json (hunk 3 above)
Comment (rmeena840) [reply infrastructure/private-net-template.json:3]: I just moved the `infrastructure` folder to here as it is. It was same there so I didn't change here.

**Comment by rmeena840** [comment]
@Megha-Dev-19 Ready for another review.

File: contracts/test/common.js (hunk 1 above)
Comment (rmeena840) [reply contracts/test/common.js]: DOne

**Review by Megha-Dev-19 — APPROVED** [review]
utACK

**Pull request merged**